Xamarin.Forms.Core/InputView.cs

```diff
@@ -1,3 +1,9 @@
+using Xamarin.Forms;
+using Xamarin.Forms.StyleSheets;
+
+[assembly: StyleProperty("-xf-inputview-max-length", typeof(InputView), nameof(InputView.MaxLengthProperty))]
+[assembly: StyleProperty("-xf-inputview-keyboard", typeof(InputView), nameof(InputView.KeyboardProperty))]
```

Comment:
just like max-length, I don't think you can set the `input` attribute in CSS (but you can select based on it)

Reply:
Again, no problemo. I only added it as it was in the spec. Did I read the spec wrong or is the spec wrong?